src/main/java/UserInfoClass.java

```diff
@@ -12,13 +12,12 @@ public class UserInfoClass {
     // Makes an api call to Spotify to get the user's top tracks over the past certain period and
     //@param timerange takes the time of how long in the past you want data for. limit is how many tracks you want to see
     // returns genre of each artist of the tracks in the data returned from Spotify
-    public static List<String> getTopTrackGenres(String timerange, String limit) throws IOException {
+    public static Map<String, Integer> getTopTrackGenres(String timerange, String limit) throws IOException {
```

Comment:
Update the Javadoc for getTopTrackGenres to reflect the new return type and purpose.